client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -292,6 +292,10 @@ const streamApis = {
 		path: ( { streamKey } ) => `/read/tags/${ streamKeySuffix( streamKey ) }/posts`,
 		dateProperty: 'date',
 	},
+	tag_popular: {
+		path: ( { streamKey } ) => `/read/tags/${ streamKeySuffix( streamKey ) }/cards`,
+		apiNamespace: 'wpcom/v2',
+	},
```

Comment:
We could add a `query` section here with a `after` param similar to discover if we want true parity. What we found is without limiting `after` to the last day or so, the feed feels very stagnant with the same top results for many days in a row.

I like this approach of updating the base stream key value you used, it feels **much cleaner** than parsing more in the suffix like we did for discover 😅